documentation/en/api-experimental-user-facing-v2-docs/api-v2-experimental.md

```diff
@@ -1107,7 +1131,7 @@ JSON-RPC allows named parameters (using a JSON object instead of an array for `p
 
 ## Future API Groups
 
-The Filecoin V2 APIs initiative will expand beyond the Chain and State API groups demonstrated in this document. Each API group will follow the same principles of using selectors for expressive queries while maintaining a minimal API surface.
+The Filecoin V2 APIs initiative will expand beyond the Chain, State, and ETH API groups demonstrated in this document. Each API group will follow the same principles of using selectors for expressive queries while maintaining a minimal API surface.
```

Comment:
I would use a casing consistent with API group name: Replace all `ETH`s with `Eth`?

Reply:
Done in https://github.com/filecoin-project/lotus/pull/13068/commits/02c2783214210999d3e26baca0892028809dfac6 .  Resolving.